test/kubectl/kubectl.go

```diff
@@ -53,7 +53,7 @@ func Apply(opts Options, arg interface{}) error {
 
 	output, err := cmd.CombinedOutput()
 
-	if logOutput {
+	if logOutput || err != nil {
```

Comment:
I've been using this many times to help debugging, I wonder if we shouldn't leave it?

Reply:
Yeah, I think it is a nice enhancement. If one of the command fails, it is nice to be able to see what failed without having to make modifications and rerun the test.